manticore/platforms/evm.py

```diff
@@ -2103,6 +2154,12 @@ def _process_pending_transaction(self):
         if failed:
             self._close_transaction('TXERROR', rollback=True)
 
+        if caller not in self._world_state:
```

Comment:
Got a little lost in the diff but the hypothesis is that this never happens, right?